.nvmrc

```diff
@@ -1 +1 @@
-v18.20.2
```

Comment:
Will this affect the CI environment? I don't think it's necessary to use node version too new here.

Reply:
> Will this affect the CI environment?

No, we specify the node version there.

> I don't think it's necessary to use node version too new here.

For copying files recursively introduced in node 22. It only requires developer's local node version.